umpleonline/scripts/umple_page.js

```diff
@@ -789,6 +797,7 @@ Page.getSelectedExample = function()
         Action.changeDiagramType({type: "GVClass"});
       }
       else {
+        // TODO: determine what behaviour should occur w/r/t JointJS
```

Comment:
Now I see why I put that off, I couldn't understand it. It looks like an unreachable branch to me.